soccer/gameplay/tests/test_ball.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+import main
+import robocup
+import evaluation.field
+import constants
+
+class Moc_Ball:
+	def __init__(self, x, y):
+		self.pos = robocup.Point(x, y)
+		self.vel = robocup.Point(0, 0)
+
+	def set_pos(self, x, y):
+		self.pos = robocup.Point(x, y)
+
+	def set_vel(self, x, y):
+		self.vel = robocup.Point(x, y)
+
+class TestBall(unittest.TestCase):
+	
+
```

Comment:
I'm not sure if `make pretty` will fix this, but to keep the style the same, the extra blank lines here to be removed.